.tito/lib/origin/builder/__init__.py

```diff
@@ -55,15 +55,37 @@ def _setup_test_specfile(self):
             )
             # Custom Openshift v3 stuff follows, everything above is the standard
             # builder
-            cmd = '. ./hack/common.sh ; echo $(os::build::ldflags)'
+            
+            ## Fixup os_git_vars
```

Comment:
You could also write a wrapper:
```sh
function os::build::emit_os_version_vars() {
    os::build::os_version_vars
    echo "OS_GIT_COMMIT=${OS_GIT_COMMIT}...."
}
```

Either way, I think it's worthwhile to keep the logic in one place so it can't drift. I found a bug in the logic this week, if there ends up a bug fix I'd rather it propagate correctly.